coin/coins.yml

```diff
@@ -422,3 +422,75 @@
   decimals: 9
   blockTime: 6000
   sampleAddress: 'oasis1qpxx577luv69znyr7dleue43mz6aq7fh4sm43cxu'
+
+- id: 22
+  symbol: MONA
+  handle: monacoin
+  name: Monacoin
+  decimals: 8
+  blockTime: 90000
+  sampleAddress: 'PSEbmFMLvjUg6vKoZBj3zPk1KVbVca5Uun'
```

Comment:
why we include coins not supported in the app?

Reply:
To have this file synced with https://github.com/trustwallet/wallet-core/blob/master/coins.json
BTW, @hewigovens could you have a look at my comment here: https://github.com/trustwallet/golibs/issues/62#issuecomment-838729521
What do you think? If we have those fields in wallet-core, we can use it to generate coins.go and no need to support the coins.yml